docs/UserGuide.md

```diff
@@ -142,6 +142,52 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding a Lesson: `ladd`
+
+Adds a lesson with the corresponding details to the specified student in the address book.
+
+**Types of lesson:**
+* Recurring
+* Makeup
+
+**Lesson fields:**
+* Date of lesson
+* dd MMM yyyy e.g. Tue, 02 Jan 2018
+* Start time HH:mm
+* End time HH:mm
+* Subject
+* Homework
+
+Format: `ladd INDEX [recurring/] date/<dd MMM yyyy> start/<HH:mm> end/<HH:mm> subject/SUBJECT [hw/HOMEWORK]`
+
+<div markdown="span" class="alert alert-primary">
+:bulb: The type of lesson will be inferred from the presence of the -recurring flag.
```

Comment:
```suggestion
:bulb: The type of lesson will be inferred from the presence of the `recurring/` flag.
```

Reply:
It seems like backticks does not work in the div. Will replacing them with double quotes be better?